### DISCUSSION TITLE
How to upgrade the `react-native-builder-bob`?

### DISCUSSION THREAD

**Discussion by susonthapa** [callstack/react-native-builder-bob#489] [Q&A]
I'm currently using version `0.18.3` and was curious how to update it or if I even need to update it. 

I tried searching through the issues and FAQs but couldn't find it. I did find the [documentation](https://callstack.github.io/react-native-builder-bob/faq#how-to-upgrade-the-react-native-version-in-the-generated-project) on upgrading the ReactNative but didn't find any migration guide for `react-native-builder-bob`.

**Comment by satya164** [comment]
Just bump the version number.
